content/22.vaccines.md

```diff
@@ -489,6 +489,31 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
 The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
 While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.
 
+### Viral evolution and vaccine protection
+
+With these vaccines in place, one concern is how long they will remain effective, especially as the virus continues to evolve.
+Since the start of this pandemic we have already seen multiple variants emerge: B.1.1.7 (emerged in the UK), B.1.351 (emerged in South Africa), P.1 (emerged in Brazil).
+
+Viruses evolve or mutate at different rates.
+Mutation rate is measured as the number of substitutions per nucleotide per cell infected (&mu_{s/n/c)) [@doi:10.1128/JVI.00694-10].
+_c_ is estimated based on the viral titers at the start of infection and after infection, and viral yield.
+_s/n_ is estimated based on the mutation frequency (number of mutations divided by the total number of nucleotides sequenced) and the total number of mutations detected by the sequencing method used.
+Given these two calculations we can estimate mutation rate. 
+RNA viruses tend to have mutation rates between 10^(-6) to 10^(-4) [@doi:10.1128/JVI.00694-10].
+As a reference, influenza A virus has a mutation rate of 10^(-5) whereas the mutation rate of SARS-CoV-2 is lower, with (10^(-6)) mutation rate [@doi:10.24875/BMHIM.20000183].
+This accumulation of mutations allows the virus to escape recognition by the immune system [@doi:10.1093/protein/gzw017].
+These mutations are usually located in epitopes, which are regions of the virual antigen (i.e. spike protein) that are bound by antibodies.
```

Comment:
I'm curious if this is a situation where most mutations are occurring in epitopes because of conservation (e.g., if it occurs elsewhere, the virus won't survive), or whether this sentence is saying that the mutations that affect immune recognition are located in epitopes

Reply:
Ah. So this was based on my still limited understanding of virology and that I know scientists tend to focus on changes in epitopes since they are those regions recognized by the immune system and therefore thought to effect immune response. I think that mutations tend to be located on the exposed (or head) regions of the antigens compared to the stalk regions. But I don't believe its known where are the key mutations that are associated with loss of immune recognition. So I think I'll remove this sentence as well. Thank you for double checking!

Reply:
I'm not sure which are associated with immune recognition issues either :/